media-gfx/viewnior/viewnior-1.7-r2.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson xdg-utils
```

Comment:
(You **can** just inherit *xdg* eclass, then you wouldn't have to define `pkg_post*` functions at all in this ebuild)

Reply:
done.